src/spacel/provision/template/app.py

```diff
@@ -213,12 +262,28 @@ def app(self, app, region):
             }
             del params['PrivateElbSubnet01'], params['PublicElbSubnet01']
             del (resources['PublicElb'], resources['PrivateElb'],
-                 resources['ElbSg'], resources['DnsRecord'],
+                 resources['ElbSg'],
                  resources['ElbHealthPolicy'],
                  resources['Asg']['Properties']['LoadBalancerNames'])
             resources['Asg']['Properties']['HealthCheckType'] = 'EC2'
             params['ElbScheme']['Default'] = 'disabled'
 
+            # Resolve the DNS record
```

Comment:
For sure. Do note that those are not free (https://aws.amazon.com/route53/pricing/) and would likely have to be opt-in, but will breadcrumb.

Would be splendid if we could have a multi region weighted DNS record automatically set up...